releasenotes/notes/ci_visibility-feat-selenium_rum_support-f1b81d0de8820aaa.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    CI Visibility: adds support for `Selenium and RUM integration <https://docs.datadoghq.com/tests/browser_tests/>`_
```

Comment:
This is the cursedest link syntax I've ever seen.

Reply:
Yes.